Tools/CISupport/ews-build/steps.py

```diff
@@ -886,6 +886,28 @@ def run(self):
         return super().run()
 
 
+class SetCredentialHelper(steps.ShellSequence, ShellMixin):
+    name = 'set-credential-helper'
+    descriptionDone = ['Set Credential Helper']
+    flunkOnFailure = False
+    haltOnFailure = False
+
+    def __init__(self, **kwargs):
+        super().__init__(timeout=5 * 60, logEnviron=False, **kwargs)
```

Comment:
Do you expect to need a 5 minutes timeout, or maybe 2 minutes is enough for this?